dev_tools/packaging/set-dev-version.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+
+# Copyright 2018 The Cirq Developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
```

Comment:
Add a documentation comment like in the other scripts.

Maybe add a message if the user doesn't `source` the script:
```
CIRQ_DEV_VERSION="$(
  set -e

  if ! (return 0 2>/dev/null); then
    echo "Usage:" >&2;
    echo "  source set-dev-version.sh" >&2;
    echo >&2;
    echo "This script sets the environment variable \$CIRQ_DEV_VERSION." >&2;
    exit 1
  fi
  ...
```

Reply:
done

Reply:
Did you add a documentation comment?

Reply:
did now (sorry I though you were asking about a comment when you misuse the script.